gradle-plugin/plugin/src/main/kotlin/com/emergetools/android/gradle/tasks/snapshots/LocalSnapshots.kt

```diff
@@ -63,15 +61,12 @@ abstract class LocalSnapshots : DefaultTask() {
 
   @get:Input
   @get:Optional
-  abstract val includePrivatePreviews: Property<Boolean>
-
-  @get:Input
-  @get:Optional
-  abstract val includePreviewParamPreviews: Property<Boolean>
```

Comment:
We can likely just remove this extension property entirely, I don't believe anyone uses it

Reply:
Did that here: https://github.com/EmergeTools/emerge-android/pull/535